packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit_webview_widget.dart

```diff
@@ -296,6 +299,27 @@ class WebKitWebViewPlatformController extends WebViewPlatformController {
     }
   }
 
+  Future<void> _setHasProgressTracking(bool hasProgressTracking) {
+    if (hasProgressTracking) {
+      webView.observeValue = (
+        String keyPath,
+        NSObject object,
+        Map<NSKeyValueChangeKey, Object?> change,
+      ) {
+        final double progress = (change[NSKeyValueChangeKey.new_] as double?)!;
```

Comment:
Nit: This can be `change[NSKeyValueChangeKey.new_]! as double` which is a bit less convoluted to read.

Reply:
Ah, I didn't know this was the alternative to `change[NSKeyValueChangeKey.new_] as double`. Thanks!